src/styles/card.scss

```diff
@@ -0,0 +1,105 @@
+@use './utils/variables.scss' as *;
+
+.card {
+  width: $width-card;
+  box-sizing: border-box;
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  transition: transform 0.3s ease-in-out;
+  transform-origin: center;
+
+  &__image {
+    display: block;
+    height: 134px;
+    width: 160px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-weight: $font-weight-500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0 0 4px;
+    color: $text-color;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    height: 14px;
+    font-family: $body-font;
+    font-size: $font-size-code;
+    line-height: 14px;
+    color: $text-color_2;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+    align-items: flex-end;
+
+    &-count {
+      font-size: $font-size-code;
+      line-height: 14px;
+      font-weight: $font-weight-400;
+    }
+  }
+
+  &__price {
+    display: flex;
+    height: 18px;
+    margin-bottom: 16px;
+    justify-content: space-between;
+
+    .size {
+      font-weight: $font-weight-400;
+      font-size: 12px;
+      line-height: 18px;
+      color: $text-color_2;
+    }
+
+    .style {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: $text-color;
+    }
```

Comment:
you shouldn't increase specificity. change size and style into card block elements and use scss nesting